contracts/human_checker/tests/workspaces.rs

```diff
@@ -171,5 +188,37 @@ async fn is_human_call() -> anyhow::Result<()> {
     tokens = suite.query_sbts(&john).await?;
     assert_eq!(tokens, None);
 
+
+    //
+    // Test Vote with lock duration
+    //
+
+    //
+    // test1: too short lock duration: should fail
+    let mut payload = VotePayload{prop_id: 10, vote: "wrong_option".to_string()};
+    let r = suite.is_human_call_lock(&john, VOTING_DURATION / 2 *3,  &payload).await?;
```

Comment:
```suggestion
    let r = suite.is_human_call_lock(&alice, VOTING_DURATION / 2 *3,  &payload).await?;
```

Reply:
`john` is not a human so it will fail in registry

Reply:
+ the voting duration is wrong check comment below

Reply:
but it will not even go to the method called by is_human_call_lock because it will fail there so the assert below will never be true